data/gegno/gegno ships.txt

```diff
@@ -0,0 +1,1135 @@
+# Copyright (c) 2022 by Saugia
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+# Gegno:
+
+ship Shale
+	sprite ship/shale
+	thumbnail thumbnail/shale
+	swizzle 0
+	attributes
+		category Transport
+		licenses
+			"Gegno Civilian"
+		cost 265000
+		mass 121
+		bunks 7
+		"cargo space" 36
+		drag 1.8
+		"engine capacity" 58
+		"fuel capacity" 400
+		"heat dissipation" 0.8
+		"heat protection" 3
+		"ion protection" 3
+		hull 760
+		"outfit space" 113
+		"required crew" 1
+		shields 0
+		"weapon capacity" 0
```

Comment:
No need to specify these attributes, for all of the relevant ships.
```suggestion
```

Reply:
Noted.